data/json/items/gun/20x66mm.json

```diff
@@ -72,6 +74,7 @@
   {
     "id": "rm228",
     "type": "GUN",
+    "looks_like": "shotgun_base",
```

Comment:
I'm fairly sure none of this will work, because I don't believe copy-from works for things defined as `abstract`

Reply:
This is correct. Ewps :P